resources/locale/en.yml

```diff
@@ -38,6 +38,12 @@ askvortsov-pwa:
         push_notif_preference_default_to_email_text: If true, users will have push notifications enabled by default for all notification types where email notifications are enabled by default. If false, users will have push notifications disabled by default.
         user_max_subscriptions_label: Max Push Subscriptions Per User
         user_max_subscriptions_text: How many push subscriptions should each user be able to have at a time? When a user receives a push notification, each subscription will result in an API call. Each subscription typically corresponds to a browser the user has used.
+        windowControlsOverlay_label: Enable Window Controls Overlay?
+        windowControlsOverlay_text: |
+          If true, users will able to enable Window Controls Overlay to create an immersive, native-like experience. Window Controls Overlay enables or not depends on user's preference and browser's compatibility. You also should be sure you've done something to match this feature.
```

Comment:
```suggestion
          If true, you will be able to provide custom CSS / JS for customizing the sizing and behavior of the native app title bar when your site is running as a PWA. The typical usecase is placing window hide / maximize / close buttons inline with your forum's top navbar, instead of above them.
```